lib/travis/logs/aggregate.rb

```diff
@@ -19,15 +19,23 @@ def setup
       def run
         loop do
           aggregate_logs
-          sleep Travis.config.logs.intervals.vacuum
+          sleep sleep_interval
         end
       end
 
       def aggregate_logs
-        Travis::Logs::Services::AggregateLogs.run
+        aggregator.run
       rescue Exception => e
         Travis::Exceptions.handle(e)
       end
+
+      private def aggregator
```

Comment:
(unless we've started doing it more inline these days? I just haven't seen it in most of our code, though I know there can be some good reasons behind it - clean code style etc)